Make.bat

```diff
@@ -1,14 +1,15 @@
 REM @ECHO off
-@SET LIT_VERSION=3.5.4
+@SET LIT_VERSION=3.7.3
+@SET LUVI_VERSION=v2.9.3-sigar
 
 IF NOT "x%1" == "x" GOTO :%1
 
 :rackspace-monitoring-agent
 ECHO "Building agent"
-IF NOT EXIST lit.exe CALL Make.bat lit
-if %errorlevel% neq 0 goto error
 IF NOT EXIST luvi-sigar.exe CALL Make.bat luvi-sigar
 if %errorlevel% neq 0 goto error
+IF NOT EXIST lit.exe CALL Make.bat lit
+if %errorlevel% neq 0 goto error

```

Comment:
why did you move these lines

Reply:
There's this weird chicken and egg thing apparently with lit and luvi:

https://github.com/luvit/lit#building-from-source

luvi is actually needed to build since you build `lit` by running its from source via luvi and at the same time provide the luvi exe to be bundled into the lit.

Before the logic was using the "get-lit" script to download a pre-built binary.